docs/rnw-dependencies.md

```diff
@@ -27,33 +27,37 @@ Start an **elevated** PowerShell window and run:
 
 Alternatively, you can setup your environment manually:
 - Ensure Developer Mode is turned ON in Windows Settings App.
-- Install [Visual Studio 2019 (version 16.5 or greater)](https://www.visualstudio.com/downloads) **with the following options checked**:
-  - Workloads
+- It is _highly_ recommended to update the Windows system.
+- Install a recent version of [Visual Studio 2019](https://www.visualstudio.com/downloads) **with the following options checked**:
+  - **Workloads**
     - Universal Windows Platform development
       - Include `C++ (v142) Universal Windows Platform tools` (under 'Optional')
+      - Older Windows 10 SDK version may be needed at this point.
     - Desktop development with C++
     - .NET Desktop development
-  - Individual Components
-    - Development activities
-      - Node.js development support (optional)
+    - Node.js development, or one of the following alternatives:
+      - Install from **Individual Components**:
+        - Development activities
+          - Node.js development support
+      - Install Node.js separately, see below for some options
+- Ensure that long path support is enabled.
 
-You will also need to ensure that certain settings are enabled:
-- Long path support
-- Developer mode enabled
-
-#### React Native Development Dependencies
-
-- Install the [standard React Native dependencies](https://reactnative.dev/docs/environment-setup)
-- Install [Node.js](https://nodejs.org) via one of the following methods:
+Options to install [Node.js](https://nodejs.org) separately:
   - Using [Chocolatey](https://chocolatey.org/) (_React Native recommended_). To use chocolatey, from an elevated Command Prompt, run:
   ```bat
   choco install nodejs-lts
   ```
+  - Using [another package manager](https://nodejs.org/en/download/package-manager/) such as [Scoop](https://scoop.sh/) or [Node Version Switcher (nvs)](https://github.com/jasongin/nvs)
```

Comment:
This line seems to cause the build failure, don't know how we should fix it.

Reply:
Markdown-spellcheck likely triggered by nvs -- if you can capitalize it, I think it will be treated as an acronym and be ignored.